trin-storage/src/sql.rs

```diff
@@ -1,71 +1,108 @@
 // SQLite Statements
+// History
+pub const CREATE_QUERY_DB_HISTORY: &str = "CREATE TABLE IF NOT EXISTS history (
+    content_id blob PRIMARY KEY,
+    content_key blob NOT NULL,
+    content_value blob NOT NULL,
+    distance_short INTEGER NOT NULL,
+    content_size INTEGER NOT NULL
+);
+    CREATE INDEX IF NOT EXISTS history_distance_short_idx ON history(content_size);
+    CREATE INDEX IF NOT EXISTS history_content_size_idx ON history(distance_short);
+";
 
-// NOTE: The indices `content_size_idx`, `content_id_short_idx` and `content_id_long_idx` weren't
-// helpful mostly because they didn't have `network` column in them.
-pub const CREATE_QUERY_DB: &str = "CREATE TABLE IF NOT EXISTS content_data (
-                                content_id_long TEXT PRIMARY KEY,
-                                content_id_short INTEGER NOT NULL,
-                                content_key TEXT NOT NULL,
-                                content_value TEXT NOT NULL,
-                                network INTEGER NOT NULL DEFAULT 0,
-                                content_size INTEGER
-                            );
-                            DROP INDEX IF EXISTS content_size_idx;
-                            CREATE INDEX IF NOT EXISTS content_size_idx_2 ON content_data(network, content_size);
-                            DROP INDEX IF EXISTS content_id_short_idx;
-                            DROP INDEX IF EXISTS content_id_short_idx_2;
-                            CREATE INDEX IF NOT EXISTS content_id_short_idx_3 ON content_data(network, content_id_short);
-                            DROP INDEX IF EXISTS content_id_long_idx;
-                            CREATE INDEX IF NOT EXISTS network_idx ON content_data(network);
-                            CREATE INDEX IF NOT EXISTS content_key_idx ON content_data(content_key);";
-
-pub const INSERT_QUERY_NETWORK: &str =
-    "INSERT OR IGNORE INTO content_data (content_id_long, content_id_short, content_key, content_value, network, content_size)
-                            VALUES (?1, ?2, ?3, ?4, ?5, ?6)";
+pub const INSERT_QUERY_HISTORY: &str =
+    "INSERT OR IGNORE INTO history (content_id, content_key, content_value, distance_short, content_size)
+                            VALUES (?1, ?2, ?3, ?4, ?5)";
```

Comment:
I want to move over to `sqlx` which doesn't support named params to my knowledge. I looked into using ``tokio_spawnblocking`` but it wasn't that clean and looked harder to maintain

Reply:
not related to this PR, but I just found this https://github.com/bikeshedder/deadpool/tree/master/r2d2 if anyone is looking. It looks like it would make it pretty drop in which is good